component/prometheus/exporter/blackbox/blackbox_test.go

```diff
@@ -38,6 +38,36 @@ func TestUnmarshalRiver(t *testing.T) {
 	require.Contains(t, "http_2xx", args.Targets[1].Module)
 }
 
+func TestUnmarshalRiverWithInlineConfig(t *testing.T) {
```

Comment:
Should we also add a test for the case that the content of `config` is invalid? I'm also curious how is this error message displayed when using this inline yaml format, and also how is displayed in combination of flow/river errors.

Reply:
Good catch! I added some test case and the make the config related error more describable. (by adding an prefix)